app/core/src/screens/MMBPackageWrapper.js

```diff
@@ -42,7 +47,8 @@ export default class MMBPackageWrapper extends React.Component<
 
   componentDidMount = () => {
     this.fetchToken();
-    this.props.navigation.setParams({ fetchToken: this.fetchToken });
+    this.props.navigation &&
+      this.props.navigation.setParams({ fetchToken: this.fetchToken });
```

Comment:
Let's clear this file and create an appropriate "StandalonePackage" file

Reply:
Should be `appRegistry` folder with root component and standalone packages.

Reply:
this file is not supposed to be a part of the native application. This is just a rapper to pass in to `ManageMyBookingPackage` what native should pass into the application